software/solution-strategy.qmd

```diff
@@ -37,7 +37,10 @@ The main technologies we will use are:
 For our decisions to use these technologies, see [Decision
 Record](https://decisions.seedcase-project.org).
 
-In relation to our [Quality Goals](introduction.qmd#quality-goals):
+<!--TODO #99 update the section on quality goals in design/solution-strategy-->
```

Comment:
You could probably just delete this section rather than comment it out.

Reply:
Do we want to leave it there for now, until someone has worked through the document? I could just leave a note saying 'consider removing'...

Reply:
@K-Beicher I don't see anything that is critical in the text that isn't covered in other parts, so deleting will keep the text clean ☺️